app/scripts/app.js

```diff
@@ -58,10 +58,12 @@ angular.module('yourStlCourts').config(function ($stateProvider, $urlRouterProvi
     })
     .state('info', {
       url: '/info',
+      controller: 'AnchorScrollCtrl as ctrl',
```

Comment:
Please change these two pages to have their own controller that use $anchorScroll instead of sharing controller and exposing service to the views directly.